airbyte-integrations/connectors/source-flexport/source_flexport/manifest.yaml

```diff
@@ -0,0 +1,158 @@
+      $ref: "#/definitions/selector"
+    paginator:
+      $ref: "#/definitions/paginator"
+    requester:
+      $ref: "#/definitions/requester"
+
+  base_stream:
+    type: DeclarativeStream
+    primary_key:
+      - id
+    retriever:
+      $ref: "#/definitions/retriever"
+
+  companies_stream:
+    $ref: "#/definitions/base_stream"
+    name: "companies"
+    $parameters:
+      path: "network/companies"
+
+  invoices_stream:
+    $ref: "#/definitions/base_stream"
+    name: "invoices"
+    $parameters:
+      path: "invoices"
+
+  locations_stream:
+    $ref: "#/definitions/base_stream"
+    name: "locations"
+    $parameters:
+      path: "network/locations"
+
+  products_stream:
+    $ref: "#/definitions/base_stream"
+    name: "products"
+    $parameters:
+      path: "products"
+
+  shipments_stream:
+    type: DeclarativeStream
+    name: "shipments"
+    primary_key:
+      - id
+    retriever:
+      type: SimpleRetriever
+      record_selector:
+        $ref: "#/definitions/selector"
+      paginator:
+        $ref: "#/definitions/paginator"
+      requester:
+        type: HttpRequester
+        url_base: https://api.flexport.com/
+        path: "shipments"
+        http_method: GET
+        request_parameters:
+          sort: updated_at
+          direction: asc
+        request_headers: {}
+        authenticator:
+          type: BearerAuthenticator
+          api_token: '{{ config[''api_key''] }}'
+        request_body_json: {}
+    incremental_sync:
+      type: DatetimeBasedCursor
+      datetime_format: '%Y-%m-%dT%H:%M:%S.%f%z'
+      cursor_field: updated_at
+      start_time_option:
+        inject_into: request_parameter
+        field_name: f.updated_at.gt
+        type: RequestOption
+      end_time_option:
+        inject_into: request_parameter
+        field_name: f.updated_at.lt
+        type: RequestOption
+      start_datetime:
+        type: MinMaxDatetime
+        datetime: '{{ config[''start_date''] }}'
+        datetime_format: '%Y-%m-%dT%H:%M:%SZ'
+      end_datetime:
+        type: MinMaxDatetime
+        datetime: '{{ now_utc().strftime(''%Y-%m-%dT%H:%M:%SZ'') }}'
+        datetime_format: '%Y-%m-%dT%H:%M:%SZ'
+
+streams:
+  - "#/definitions/companies_stream"
+  - "#/definitions/invoices_stream"
+  - "#/definitions/locations_stream"
+  - "#/definitions/products_stream"
+  - "#/definitions/shipments_stream"
+
+spec:
+  connection_specification:
+    $schema: http://json-schema.org/draft-07/schema#
+    type: object
+    required:
+      - api_key
+      - start_date
+    properties:
+      api_key:
+        type: string
+        title: API Key
+        airbyte_secret: true
+        order: 0
+      start_date:
+        type: string
+        title: Start date
+        format: date-time
+        pattern: ^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z$
+        order: 1
+
+    additionalProperties: true
+  documentation_url: "https://docs.airbyte.com/integrations/sources/flexport"
+  type: Spec
```

Comment:
Can you format this?

Reply:
Can you clarify what you mean by format?

Reply:
```yaml
spec:
  type: Spec
  documentation_url: "https://docs.airbyte.com/integrations/sources/flexport"
  connection_specification:
    $schema: http://json-schema.org/draft-07/schema#
    type: object
    additionalProperties: true
    required:
      - api_key
      - start_date
    properties:
      api_key:
        type: string
        title: API Key
        airbyte_secret: true
        order: 0
      start_date:
        type: string
        title: Start date
        format: date-time
        pattern: ^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z$
        order: 1